### PR TITLE
util: add some debug log for StartCPUProfile and StopCPUProfile | tidb-test=2d8288cf58aec9d381d09da70b18fd27855ed597

### PR DESCRIPTION
**Pull request by XuHuaiyu**
### What problem does this PR solve?

Issue Number: ref #53958 
As the comment in the issue says

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by crazycs520 — DISMISSED** [review]

File: util/cpuprofile/cpuprofile.go [hunk 1]
```diff
@@ -157,7 +157,9 @@ func (p *parallelCPUProfiler) profilingLoop() {
 	checkTicker := time.NewTicker(DefProfileDuration)
 	defer func() {
 		checkTicker.Stop()
+		logutil.BgLogger().Info("before StopCPUProfile")
```
Comment (zimulala) [inline util/cpuprofile/cpuprofile.go:160]: Would "start StopCPUProfile" and "finish StopCPUProfile" be better？ Feel like these "before" and "after" are rarely used?

**Review by lcwangchao — DISMISSED** [review]

**Review by lcwangchao — APPROVED** [review]

**Review by crazycs520 — APPROVED** [review]

**Review by wshwsh12 — APPROVED** [review]

**Pull request merged**